x/auth/ante/fee.go

```diff
@@ -81,25 +83,26 @@ func (dfd DeductFeeDecorator) checkDeductFee(ctx sdk.Context, sdkTx sdk.Tx, fee
 	}
 
 	feePayer := feeTx.FeePayer()
-	feeGranter := feeTx.FeeGranter()
+	// feeGranter := feeTx.FeeGranter()
```

Comment:
```suggestion
	feeGranter := []byte(nil) // feeTx.FeeGranter()
```

In case you want to disable the code below with minimum changes

Reply:
Thanks!

Reply:
Resolved